scripts/update-version.cjs

```diff
@@ -0,0 +1,20 @@
+const GITHUB_API_URL = "https://api.github.com/repos/kuzudb/kuzu/releases/latest";
+const fs = require('fs/promises');
+const path = require('path');
+
+(async () => {
+    console.log("Fetching latest version from GitHub...");
+    const response = await fetch(GITHUB_API_URL);
```

Comment:
We should have a way to manually specify the version, as this script should be run as part of the `dev` to `main` release merge, which could be before the actual version has been released.

Reply:
I have changed the behavior:
- For the deployment pipeline, there is a way to skip version updating, so that the build process uses the version specified in JSON directly.
- For dev docker deployment, always use the manually specified version (not doing auto version update).

Since now it is just a JSON file that can easily be edited directly. Using the script or workflow to do version update is mainly for automatically version maintaining but not necessary.